apps/web/src/components/tx/ConfirmTxDetails/TxDetails.tsx

```diff
@@ -185,3 +195,27 @@ export const TxDetails = ({ safeTx, txData, showHashes }: TxDetailsProps) => {
     </PaperViewToggle>
   )
 }
+
+function TxDetailsHashes({ safeTx }: Pick<TxDetailsProps, 'safeTx'>) {
+  const safeTxHash = useSafeTxHash({ safeTxData: safeTx.data })
+  const domainHash = useDomainHash()
+  const messageHash = useMessageHash({ safeTxData: safeTx.data })
+
+  return (
+    <>
+      {[
+        ['Domain hash', domainHash] as const,
+        ['Message hash', messageHash] as const,
+        ['safeTxHash', safeTxHash] as const,
+      ].map(([label, hash]) => {
+        return (
+          <TxDetailsRow label={label} key={hash}>
+            <Typography variant="body2" width="100%" sx={{ wordWrap: 'break-word' }}>
```

Comment:
`mb={-0.5}`

Reply:
Looking back at the design, my initial commit was incorrect. I adjusted the implementation slightly to include `Divider`s between each hash in 8088f81565c50bbda19222d8b920bd696b365acd. What do you think of the following (without a margin adjustment)?

![image](https://github.com/user-attachments/assets/9a8dcd94-23f2-4f86-93c0-61dfd6117072)

Reply:
Looks good! Could you also add `padding-left: 0` to the copy button in HexEncodedData? (There's already CSS for it).

Reply:
Here's the CSS: https://github.com/safe-global/safe-wallet-monorepo/blob/dev/apps/web/src/components/transactions/HexEncodedData/styles.module.css#L7-L10